www/kubl05/homework-05/main.js

```diff
@@ -0,0 +1,174 @@
+/* HOMEWORK NUMERO CINCO */
+/**
+ * Pepe's age. Vypište na konzoli smysluplnou oznamovací větu ohledně věku Pepy, pokud znáte jeho rok narození. 
+ * Použijte proměnné a pro výpis použijte zřetězení stringů. Jako názvy proměnných používejte anglické pojmy.
+ */
+// SPOILER ALERT!
+
+var birthYear = 2000;
+
+var currentYear = new Date().getFullYear();
+var age = currentYear - birthYear;
+
+console.log("Pepeho věk je: " + age);
+
+/**
+ * WTF (wow, that's fun). Vypište teplotu v Fahrenheiht, pokud znáte teplotu v Celsius, a také naopak. 
+ * Formát výpisu je: 20°C =  68°F resp. 68°F = 20°C. Výpočet probíhá takto:
+ *     z C na F: vynásobit devíti, vydělit pěti a přičíst 32. 
+ *     z F na C: odečíst 32, vynásobit pěti a vydělit devítkou. 
+ */
+// SPOILER ALERT!
+
+var celsius = 20, fahrenheiht = 100;
+var fahrenheihtTransferred, celsiusTransferred;
+
+fahrenheihtTransferred = ((celsius * 9) / 5) + 32;
+celsiusTransferred = ((fahrenheiht - 32) * 5) / 9;
+
+console.log(celsius + "°C = " + fahrenheihtTransferred + "°F");
+console.log(fahrenheiht + "°F = " + celsiusTransferred + "°C");
+
+/**
+ * Funkce function fonction funktio. Vemte předchozí úlohy a udělejte z nich funkce. Tj. vytvořte funkce, 
+ * které přijímají argumenty, a na základě argumentů po zavolání vypíše výsledek na konzoli. 
+ * Párkrát zavolejte tyto funkce s různými argumenty. V konzoli také vyzkoušejte, zda fungují vaše funkce. 
+ */
+// SPOILER ALERT!
+
+var getAge = function (birthYear) {
+    var currentYear = new Date().getFullYear();
+    var age = currentYear - birthYear;
+
+    return console.log("Pepeho věk je: " + age);
```

Comment:
`return` se pouziva pro vraceni nejake hodnoty, v tomto pripade tato funkce nevraci nic, takze `return` tady nema co delat. 

pripadne lze vratit hodnotu `age`, ale ne to vypisovani

podobne u vsech ostatnich funkci